client/ayon_core/tools/sceneinventory/models/containers.py

```diff
@@ -149,26 +151,35 @@ def new_invalid(cls):
 
 
 class VersionItem:
-    def __init__(self, version_id, product_id, version, status, is_latest):
-        self.version = version
-        self.version_id = version_id
-        self.product_id = product_id
-        self.version = version
-        self.status = status
-        self.is_latest = is_latest
+    def __init__(
+        self,
+        version_id: str,
+        product_id: str,
+        version: int,
+        status: str,
+        is_latest: bool,
+        is_last_approved: bool,
+    ):
+        self.version_id: str = version_id
+        self.product_id: str = product_id
+        self.version: int = version
+        self.status: str = status
+        self.is_latest: bool = is_latest
+        self.is_last_approved: bool = is_last_approved
 
     @property
     def is_hero(self):
         return self.version < 0
 
     @classmethod
-    def from_entity(cls, version_entity, is_latest):
+    def from_entity(cls, version_entity, is_latest, is_last_approved):
         return cls(
             version_id=version_entity["id"],
             product_id=version_entity["productId"],
             version=version_entity["version"],
             status=version_entity["status"],
             is_latest=is_latest,
+            is_last_approved=is_last_approved,
```

Comment:
Is it odd that we call the state `done` but here refer to it as `approved`. Should we rename this `is_last_done`?
Admittedly, that's a confusing name - but at least it's referring to the same thing. Or, we refactor the status name from `done` to `approved` so it matches with this?

Reply:
It should really care about last approved, how the value is filled might change in future.

NOTE: We had a long discussion about how to define "last approved" and came to conclusion that it might change in future with more specific settings.